firstvoices/backend/admin/characters_admin.py

```diff
@@ -50,20 +60,7 @@ class CharacterRelatedDictionaryEntryInline(BaseInlineAdmin):
     fields = ("character", "dictionary_entry")
     readonly_fields = BaseInlineAdmin.readonly_fields
     can_delete = True
-
-    # see fw-4234 about filtering the dictionary entries by site here
-    def formfield_for_foreignkey(self, db_field, request, **kwargs):
-        if db_field.name == "dictionary_entry":
-            # Get the Character from the request and filter dictionary entries by its site_id
-            character_id = request.resolver_match.kwargs.get("object_id")
-            if character_id:
-                queryset = DictionaryEntry.objects.filter(
-                    site_id=self.model.character.field.related_model.objects.get(
-                        pk=character_id
-                    ).site_id
-                )
-                kwargs["queryset"] = queryset
-        return super().formfield_for_foreignkey(db_field, request, **kwargs)
+    autocomplete_fields = ("dictionary_entry",)
```

Comment:
[minor] During local testing the related dictionary entry field was not filtered by site and other site's entries could be selected as related entries. May not be a concern as adding related entries to character will mostly be a front end action.

Reply:
I have added filtering for this. Thanks!